certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -2454,8 +2379,8 @@
               "properties": {
                 "boardid": {
                   "type": "string",
-                  "description": "boardid",
-                  "title": "Board",
+                  "description": "-- select board --",
```

Comment:
@kostadin-kanbanize please make sure the description looks like a sentence. If there's a dropdown list, you can ask the user to select the value, e.g. `"Select a board from the dropdown list"`

Reply:
Description updated